src/cmd/flux-mini.py

```diff
@@ -238,8 +238,9 @@ def create_parser(self):
             "-o",
             "--setopt",
             action="append",
-            help="Set shell option (multiple use OK)",
-            metavar="KEY=VAL",
+            help="Set shell option OPT. An optional value is supported with"
+            + " OPT=VAL (default=1). This option may be used multiple times",
```

Comment:
Minor: maybe change wording to match setattr long description?

Specifically, use "(multiple use OK)" instead of "This option may be used multiple times"?  Or change the other one if that's too terse.

Reply:
Oh yeah, I didn't want to update an unrelated option in one commit. I find the "multiple use OK" a bit unnecessarily terse and possibly confusing to the common user, but could go either way.